src/tests/unit/tests/injected/visualization/highlight-visualization.test.tsx

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import { IMock, Mock } from 'typemoq';
+
+import { FeatureFlagStoreData } from '../../../../../common/types/store-data/feature-flag-store-data';
+import { WindowUtils } from '../../../../../common/window-utils';
+import { RenderDialog } from '../../../../../injected/dialog-renderer';
+import { AxeResultsWithFrameLevel } from '../../../../../injected/frameCommunicators/html-element-axe-results-helper';
+import { DrawerUtils } from '../../../../../injected/visualization/drawer-utils';
+import { DrawerConfiguration, Formatter } from '../../../../../injected/visualization/formatter';
+import {
+    HighlightVisualization,
+    HighlightVisualizationDeps,
+    HighlightVisualizationProps,
+} from '../../../../../injected/visualization/highlight-visualization';
+import { NodeListBuilder } from '../../../common/node-list-builder';
+
+describe('HighlightVisualization', () => {
+    let drawerUtilsMock: IMock<DrawerUtils>;
+    let windowUtilsMock: IMock<WindowUtils>;
+    let formatterMock: IMock<Formatter>;
+    let documentMock: IMock<Document>;
+    let featureFlagStoreDataStub: FeatureFlagStoreData;
+    let renderDialogStub: RenderDialog;
+    let deps: HighlightVisualizationDeps;
+    let props: HighlightVisualizationProps;
+    let elementResults: AxeResultsWithFrameLevel[];
+    let bodyStub: HTMLBodyElement;
+    let bodyStyleStub: CSSStyleDeclaration;
+    let docStyleStub: CSSStyleDeclaration;
+    let documentElementStub: HTMLElement;
+
+    beforeEach(() => {
+        drawerUtilsMock = Mock.ofType(DrawerUtils);
+        windowUtilsMock = Mock.ofType(WindowUtils);
+        formatterMock = Mock.ofType<Formatter>();
+        documentMock = Mock.ofType<Document>();
+        featureFlagStoreDataStub = {};
+        renderDialogStub = () => {};
+        elementResults = [];
+        bodyStyleStub = {
+            height: '1',
+        } as CSSStyleDeclaration;
+        docStyleStub = {
+            width: '1',
+        } as CSSStyleDeclaration;
+        documentElementStub = {} as HTMLElement;
+        bodyStub = {} as HTMLBodyElement;
+
+        drawerUtilsMock.setup(mock => mock.getDocumentElement()).returns(() => documentMock.object);
```

Comment:
calling `mock` a `mock` seems as generic as it can gets... care to be a little more specific?